---
layout: single
classes: wide
permalink: /
author_profile: true


title: "Ninad Oak, Ph. D."
excerpt: "Genetics, Sketchnotes, and more..."
sidebar:
  - title: "Ninad Oak"
    image: /assets/images/Profile2-small.jpg



---

After completing my PhD in Genetics at Baylor College of Medicine, I joined St. Jude Children's Research Hospital as a Bioinformatics Research Scientist.
The focus of my current work is to discover genetic basis of childhood cancer predisposition.
During my PhD, I developed [bioinformatic approaches (ADmiRE)](https://github.com/nroak/admire) to understand the impact of microRNA single nucleotide variation in human population and disease datasets.

## Research Interests

- Cancer predisposition in childhood and adult cancers.
- Understanding the role of non-coding RNA (microRNA) sequence variation in human population and diseases.

In addition to doing science, I love to live-sketch scientific talks. You can find all of my picture summaries [here](sketchnotes).

---
## Publications

- [PubMed Bibliography](https://www.ncbi.nlm.nih.gov/sites/myncbi/1jQDtZLm9on5T/bibliography/40122621/public/?sort=date&direction=ascending)
- [Google Scholar](https://scholar.google.com/citations?user=oE-Da48AAAAJ&hl=en)

An updated list of published and unpublished papers is listed in reverse chronological order.

- **Oak N**, Ghosh R, Huang KL, Wheeler DA, Ding L, Plon SE. Framework for microRNA variant annotation and prioritization using human population and disease datasets. Human Mutation. 2019; 40(1):73-89 [PMID: 30302893](https://www.ncbi.nlm.nih.gov/pubmed/30302893)
- Huang KL, Mashl RJ, Wu Y, Ritter DI, Wang J, Oh C, Paczkowska M, Reynolds S, Wyczalkowski MA, **Oak N**, ..., Plon SE, Chen F, Ding L. Pathogenic Germline Variants in 10,389 Adult Cancers. Cell. 2018; 173(2):355-370.e14. NIHMSID: NIHMS957308 [PMID: 29625052](https://www.ncbi.nlm.nih.gov/pubmed/29625052)
- Ghosh R, **Oak N**, Plon SE. Evaluation of in silico algorithms for use with ACMG/AMP clinical variant interpretation guidelines. Genome biology. 2017; 18(1):225. [PMID: 29179779](https://www.ncbi.nlm.nih.gov/pubmed/29179779), [PMCID: PMC5704597](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5704597/)
- Chandrani P, Upadhyay P, Iyer P, Tanna M, Shetty M, Raghuram GV, **Oak N**, Singh A,  Chaubal R, Ramteke M, Gupta S, Dutt A. Integrated genomics approach to identify biologically relevant alterations in fewer samples. BMC genomics. 2015; 16:936. [PMID: 26572163](https://www.ncbi.nlm.nih.gov/pubmed/26572163), [PMCID: PMC4647579](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4647579/)
- **Oak N**, Jayaraman VK. Identification of ligand binding pockets on nuclear receptors by machine learning methods. Protein and peptide letters. 2014; 21(8):808-14. [PMID: 23855664](https://www.ncbi.nlm.nih.gov/pubmed/23855664)

---


## ADmiRE: Annotative Database of miRNA Elements
### ADmiRE Highlights

- Annotation wrapper for adding comprehensive miRNA annotations to a user supplied list of variants (tab-separated format)
- Adds information for miRNA domains, gnomAD mean allele frequency percentiles, evolutionary conservation, etc.

### GitHub

- [Download from GitHub](https://github.com/nroak/ADmiRE)

### Published in Human Mutation

- *Framework for microRNA variant annotation and prioritization using human population and disease datasets* [Human Mutation](https://onlinelibrary.wiley.com/doi/full/10.1002/humu.23668)

---

## Scientific coverage:

- [PLOS ECR blog coverage](http://blogs.plos.org/thestudentblog/2017/01/31/the-art-of-selling-science-presenting-an-engaging-scientific-talk/) - A blog by Caitlin Grzeskowiak highlighting my sketchnote style of note-taking.
