# NINAD R. OAK, M.S.

Ninad.Oak@bcm.edu; ninadoak@gmail.com 832-571-8017

## Education

### PhD, Baylor College of Medicine , Houston TX        
_2012- expected 2018_ <br >

Dept. of Molecular and Human Genetics, Graduate School of Biomedical Sciences<br >

**Thesis title:** Uncovering the landscape of miRNA sequence variation and conservation from human population and adult cancers

**Thesis mentor:** Sharon E. Plon, MD, PhD

### M.Sc. in Bioinformatics, University of Pune, Pune, India
_2009-2011_

**Thesis title:** Classification of nuclear receptor ligand binding pockets using machine learning methods.

**Thesis mentor:** V. K. Jayaraman, PhD, Center for Development of Advanced Computing (C-DAC), India

###B.Sc. in Medical Biotechnology, Pravara Institute of Medical Sciences, India 

_2006-2009_


### Biomedical Collaboration and Teamwork Certification, _2017_



## Honors and Awards:

- 3rd place for poster at 25th Anniversary Texas Children&#39;s Research Symposium, _2018_
- 2nd place for poster at Annual Texas Children&#39;s Research Symposium, _2015_
- BioInformatics National Certification (BINC), Govt. of India, All India Rank 5, _2011_
- N. Ramachandran Fellowship, Govt. of India, _2009-2011_
- Department of Biotechnology Academic Fellowship, Govt. of India, _2009-2011_

## Work Experience

### PhD Student/ Graduate Research Assistant – Plon laboratory, _2012- Present_

**Baylor College of Medicine, Houston, TX**

- Designed annotation platform and completed the analysis of miRNA sequence variation from human population sequencing datasets (gnomAD, ExAC, 1000G, etc.) and cancer datasets (TCGA PanCancerAtlas, St. Jude&#39;s PCGP, etc.).
- Analysis of missense variant deleteriousness prediction tools using ClinVar dataset
- Analysis of rare germline variation from germline TCGA PanCancerAtlas, familial leukemia to identify potential cancer predisposition genes.
- Implementation various bioinformatic tools for variant calling (GATK Best Practices, MUTECT2, Platypus), variant annotation tools (VEP, ANNOVAR, Oncotator), and programing in Perl, Python, Linux, and R/Bioconductor environment.
- Performed _in vitro_ functional assays to assess effect of mutations on miRNA activity.

### Research Rotations, _2012-2013_

**Baylor College of Medicine, Houston, TX**

- Gad Shaulsky Laboratory: RNAseq analysis to identify top differentially expressed genes from _Dictyostelium discoideum_ strains responsible for social behavior.
- Penelope Bonnen Laboratory: Analysis of _MSH1, MSH2_ and _MLH1_ genes for _de novo_ mutations in a Lynch syndrome cohort.
- Kenneth Scott Laboratory: Functional analysis of cancer driver genes using gateway cloning.

### Junior Research Fellow, _2011- 2012_

**Advanced Center for Treatment, Research, and Education in Cancer (ACTREC), Navi Mumbai, India**

- Initiated and led the project &quot;Analysis of DNA genotyping data from head and neck cancer cell-lines to identify disease causal genes&quot;; presented work at GCGC &#39;11 international cancer conference.
- Designed and optimized analysis workflow to identify candidate copy number variation using Illumina GenomeStudio and performed validation using karyotyping.

### M.Sc. Thesis project, _2011_

**Bioinformatics Center, University of Pune, India**

- Designed and executed a scientific thesis project by independently developing research hypothesis, research plan and carrying out data analysis.
- Developed a machine learning prediction tool for the prediction of nuclear receptor ligand binding pockets through protein structure analysis using programmatic approach.

### Internship, _Summer__2008_

**Ethypharm LLC Pvt. Ltd, Mumbai, India**

- Assessed quality management system for 18 different pharmaceutical compounds.

## Professional Memberships and Certifications:

- Member of American Society of Human Genetics, _2017- Present_
- Life Science Entrepreneurship Certification, Rice University, _2017_
- Member of American Society of Biochemistry and Molecular Biology, _2017- Present_

## Professional Activities:

Experience of leading many cross-functional, inter-disciplinary teams in academic and non-academic settings.

- Chair and Student Ambassador, Alumni Student Ambassador Board, Baylor College of Medicine, _2016- Present_

- President, Indian Students and Scholars Association (ISSA), Baylor College of Medicine, _2014- 2015_

- Genetics Representative, Graduate Student Council, Baylor College of Medicine, _2013- 2015_

- Organizing Committee, 1st Global Cancer Genomics Consortium, _2011_
- Founded and Chaired, Graduate Student Council, University of Pune, _2009- 2011_

## Mentorship and Teaching:

### Graduate and Undergraduate Trainees

- Saumya Sisoudiya; Plon Lab graduate student

- Adam Hansen; MHG rotation student

- Varuna Chandar; MHG rotation student

- Justin Branch; BCM-SMART program student

- Esther Akinwande; BCM-SMART program student

### Teaching

- Guest Lecturer for Undergraduate class (senior year) of Bioinformatics, Pillai College of Arts, Science, and Commerce, Panvel, India, _2011- 2012_



## Bibliography:

### Research Papers (Peer Reviewed)

**1.** Huang KL, Mashl RJ, Wu Y, Ritter DI, Wang J, Oh C, Paczkowska M, Reynolds S, Wyczalkowski MA, **Oak N, …,** Plon SE, Chen F, Ding L (2018)Pathogenic Germline Variants in 10,389 Adult Cancers. _Cell_. 173: 355–370

**2.** Ghosh R, **Oak N** , Plon SE. (2017) Evaluation of in silico algorithms for use with ACMG/AMP clinical variant interpretation guidelines. _Genome Biology._ 18(1):225.

**3.** Chandrani P, Upadhyay P, Iyer P, Tanna M, Shetty M, Raghuram GV, **Oak N** , Singh A, Chaubal R, Ramteke M, Gupta S, Dutt A. (2015) Integrated genomics approach to identify biologically relevant alterations in fewer samples. _BMC Genomics._ 16:936.

**4. Oak N**, Jayaraman VK. (2014) Identification of ligand binding pockets on nuclear receptors by machine learning methods. _Protein Peptide Lett_. 21(8):808-14.

### Research Papers (To be resubmitted)

**1. Oak N**, Ghosh R, Huang KL, Ding L, Wheeler DA, Plon SE, Annotative Database of miRNA Elements, ADmiRE, uncovers miRNA variation in human population and cancers. [bioRxiv:177170](https://www.biorxiv.org/content/early/2017/08/16/177170).

### Conference Presentations (Selected first author)

**1. Oak N**, Ghosh R, Huang KL, Ritter DI, Wheeler DA, Ding L, Plon SE. (2018).ADmiRE—Annotation of microRNA sequence variation across human population and adult cancer datasets. Presented (Poster) at the Biology of Genomes, Cold Spring Harbor Laboratories, New York.

**2. Oak N**, Ghosh R, Huang KL, Ritter DI, Wheeler DA, Ding L, Plon SE. (2018).ADmiRE—Annotation of microRNA sequence variation across human population and adult cancer datasets. Presented (Poster) at 25th Anniversary Texas Children&#39;s Research Symposium, Houston, TX

**3. Oak N**, Ghosh R, Plon SE. (2017). Annotative Database of miRNA Elements, ADmiRE, identifies the landscape of human miRNA variation from the genome aggregation dataset, gnomAD. Presented at the Gordon Research Seminar (Talk) and Conference (Poster) on Human Genetics and Genomics, Stowe, Vermont.

**4. Oak N**, Plon SE. (2015). Defining the microRNA mutational landscape in 1000 Genomes and pediatric acute lymphocytic leukemia datasets. Presented (Poster) at the Biology of Genomes, Cold Spring Harbor Laboratories, New York.

**5. Oak N**, Plon SE. (2015). Defining the microRNA mutational landscape in 1000 Genomes and pediatric acute lymphocytic leukemia datasets. Presented (Poster) at Annual Texas Children&#39;s Research Symposium, Houston, TX.

**6. Oak N,** Ritter DI, Powell BC, Cheung HC, Gramatges MM, Wheeler WA, Plon SE. (2014). Identification of germline mutations in _TEP1_ among familial and sporadic pediatric acute myelogenous leukemia cohorts. Presented (Poster) at the American Society of Human Genetics, San Diego, CA.

**7. Oak N**, Upadhyay P, Chaubal R, Chandrani P, Dutt A. (2011). Integrated analysis of rare HNSCC cell-lines using high resolution SNParrays and next generation sequencing. Presented (Poster) at the Global Cancer Genomics Consortium, Navi Mumbai, India.

**8. Oak N**, Sharma S, Sundararajan V, Jayaraman VK (2011). Application of machine learning methods to identify nuclear receptor ligand binding pockets. (Poster) InCoB/ISCB-Asia, Kuala Lumpur, Malaysia.